utils/data/demos.js

```diff
@@ -165,6 +165,18 @@ const DEMOS = [
       twitter: 'tapasadhikary',
       email: 'tapas.adhikary@gmail.com',
     },
+  },
+  {
+    id: '_screen_capture_',
+    title: '💻 Screen Capture',
+    desc: 'The Screen Capture demo introduces additions to the existing Media Capture and Streams API to let the user select a screen to capture as a media stream',
+    path: 'screen-capture',
+    canIUseURL: 'https://caniuse.com/',
```

Comment:
It should point to the screencapture API of can I use

Reply:
Done @atapas